### DISCUSSION TITLE
"iPhone-view"

### DISCUSSION THREAD

**Discussion by ChrisGitHub42** [Zefau/ioBroker.jarvis#2823] [Ideas / Feature Requests]
I'd like to create a view in Jarvis, which I can setup similar as the classical iPhone view, 3-4 icons in a row with a small text below, clickable, that it jumps direct on the page in the jarvis view...

I was able to create something "in this direction" with HomeKitTile, but there is the issue, that I see always an error message, "HomeKitTile: Device undefined not found"
--> yes of course, I do not want a device there... I just want the clickable Icon, what is already working...

What I wasn't able to do so fare, is to add a text direct below the icon, similar as on an iPhone... and test, if the ordering really works, so I have 3-4 of this icon on one line... Reason I cannot see/verify this so far is the wide error message....

Background is, that I want a "home-page" to access on iPhone, where we can reach the most important view with one click... Currently we often have to first select the "main-page" on the menu on the left, and than brows through the different tabs...

**Comment by mcuiobroker** [comment]
You need a Device, so use a Dummy Device.
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/module/homekittile/main-dashboard-hkt
